### PR TITLE
Update upgrade.md

### PR DESCRIPTION
**Pull request by YiyunNi**
Updates include:
1. Replace the example in this tutorial.
2. Restructured the headings 
3. Stylistic changes to follow the Milvus doc style guide.

### CODE REVIEW

**Comment by YiyunNi** [comment]
This PR will close issue #941 once merged,

**Review by claireyuw — CHANGES_REQUESTED** [review]

File: site/en/adminGuide/upgrade.md [hunk 1]
```diff
@@ -6,187 +6,240 @@ summary: Learn how to upgrade Milvus.
...
 NAME              NAMESPACE        REVISION        UPDATED                                     STATUS          CHART               APP VERSION
-my-release        default          1               2021-09-06 14:46:33.920893 +0800 CST        deployed        milvus-2.1.5        2.0.0-rc.4
+my-release        default          1               2021-11-08 17:12:44.678247 +0800 CST        deployed        milvus-2.2.4        2.0.0-rc.7
 ```
 
-You can see the `APP VERSION` is **2.0.0-rc4**.
+### 2. Check the running pods
 
-2. Check the running pods:
+Run `$ kubectl get pods` to check the running pods. You can see the following output.
```
Comment (chris-zilliz) [inline site/en/adminGuide/upgrade.md:26]: Do you need "If successful?"

File: site/en/adminGuide/upgrade.md [hunk 2]
```diff
@@ -6,187 +6,240 @@ summary: Learn how to upgrade Milvus.
...
-my-release-minio-5564fbbddc-dw77v               1/1     Running   0          110s
+my-release-etcd-0                               1/1     Running   0          84s
+my-release-milvus-standalone-75c599fffc-6rwlj   1/1     Running   0          84s
+my-release-minio-744dd9586f-qngzv               1/1     Running   0          84s
 ```
 
-3. Check the image tag for the pod `my-release-milvus-standalone-66f985d5cd-q5qhj`:
+### 3. Check the image tag
 
+Check the image tag for the pod `my-release-milvus-standalone-75c599fffc-6rwlj`. You can see your Milvus standalone version is v2.0.0-rc7.
```
Comment (chris-zilliz) [inline site/en/adminGuide/upgrade.md:37]: Do you need "If successful?"

File: site/en/adminGuide/upgrade.md [hunk 3]
```diff
@@ -6,187 +6,240 @@ summary: Learn how to upgrade Milvus.
...
+milvusdb/milvus:v2.0.0-rc7-20211011-d567b21
+```
+
 
-You can see the Milvus standalone version is **v2.0.0-rc4**.
+### 4. Check new Milvus standalone versions
 
-4. Run the following commmand to check new Milvus versions:
+Run the following commands to check new Milvus versions. You can see there are several new versions after v2.0.0-rc7. 
+
```
Comment (chris-zilliz) [inline site/en/adminGuide/upgrade.md:51]: Same comment as above. Up to you to decide.

File: site/en/adminGuide/upgrade.md [hunk 4]
```diff
@@ -6,187 +6,240 @@ summary: Learn how to upgrade Milvus.
...
+milvus/milvus        2.2.2                2.0.0-rc.7                Milvus is an open-source vector database built ...
+milvus/milvus        2.2.1                2.0.0-rc.6                Milvus is an open-source vector database built ...
+milvus/milvus        2.2.0                2.0.0-rc.6                Milvus is an open-source vector database built ...
 ```
 
-You can see there are several new versions after **v2.0.0-rc4**.
+### 5. Upgrade
 
-5. Upgrade to **v2.0.0-rc5-hotfix1**:
+Run the following commands to upgrade your Milvus version from v2.0.0-rc7 to v2.0.0-rc8.
```
Comment (chris-zilliz) [inline site/en/adminGuide/upgrade.md:74]: As Claire said, please capitalize "RC."

File: site/en/adminGuide/upgrade.md [hunk 5]
```diff
@@ -6,187 +6,240 @@ summary: Learn how to upgrade Milvus.
...
 
 ```
-helm repo update
-helm upgrade my-release milvus/milvus --set cluster.enabled=true
-NAME              NAMESPACE        REVISION        UPDATED                                     STATUS          CHART                APP VERSION
-my-release        default          2               2021-09-06 16:18:40.021412 +0800 CST        deployed        milvus-2.1.14        2.0.0-rc.5-hotfix1
+$ helm repo update
+$ helm upgrade my-release milvus/milvus --set cluster.enabled=true
```
Comment (chris-zilliz) [inline site/en/adminGuide/upgrade.md:197]: Do we still need "--set cluster.enabled=true?"

File: site/en/adminGuide/upgrade.md [hunk 6]
```diff
@@ -6,187 +6,240 @@ summary: Learn how to upgrade Milvus.
...
-helm list
-NAME              NAMESPACE        REVISION        UPDATED                                     STATUS          CHART                APP VERSION
-my-release        default          2               2021-09-06 15:01:24.570561 +0800 CST        deployed        milvus-2.1.14        2.0.0-rc.5-hotfix1
+$ helm repo update
+$ helm upgrade my-release milvus/milvus --set cluster.enabled=false --set etcd.replicaCount=1 --set minio.mode=standalone --set pulsar.enabled=false 
 ```
 
-You can see the new pods:
+Run `$ helm list` again to check your Milvus version. You can see your Milvus standalone has been updated to v2.0.0-rc8.
+
```
Comment (chris-zilliz) [inline site/en/adminGuide/upgrade.md:82]: "RC." Please change globally.

File: site/en/adminGuide/upgrade.md [hunk 7]
```diff
@@ -6,187 +6,240 @@ summary: Learn how to upgrade Milvus.
...
-my-release-milvus-standalone-546649bcdf-xqjd5   1/1     Running   0          31m
-my-release-minio-744dd9586f-drjnr               1/1     Running   0          31m
+my-release-etcd-0                               1/1     Running   0          3m32s
+my-release-milvus-standalone-6967454987-72r55   1/1     Running   0          22s
+my-release-minio-744dd9586f-qngzv               1/1     Running   0          3m32s
 ```
 
 <div class="alert note">
-To upgrade Milvus standalone, old pods will be deleted first. Therefore, the service may be offline for a short period of time.
+When upgrading Milvus standalone version, old pods will be deleted. Therefore, the service may be offline for a short period of time.
```
Comment (chris-zilliz) [inline site/en/adminGuide/upgrade.md:98]: I don't think you need "version" with "upgrading."
"When upgrading Milvus standalone" is good enough. 
Up to you anyway.

File: site/en/adminGuide/upgrade.md [hunk 8]
```diff
@@ -6,187 +6,240 @@ summary: Learn how to upgrade Milvus.
...
 ```
-kubectl get pods my-release-milvus-proxy-84dcb766c9-l8srs -o=jsonpath='{$.spec.containers[0].image}'
-milvusdb/milvus:v2.0.0-rc4-20210811-bdb8396
+milvusdb/milvus:v2.0.0-rc7-20211011-d567b21
 ```
 
-You can see the version of Milvus cluster is **v2.0.0-rc4**.
+### 4. Check new Milvus cluster versions
+
+Run the following commands to check new Milvus versions. You can see there are several new versions after 2.0.0-rc7.
```
Comment (chris-zilliz) [inline site/en/adminGuide/upgrade.md:169]: > new versions **later than** 2.0.0-**RC7**?

File: site/en/adminGuide/upgrade.md [hunk 9]
```diff
@@ -6,187 +6,240 @@ summary: Learn how to upgrade Milvus.
...
-helm repo update
-helm upgrade my-release milvus/milvus --set cluster.enabled=true
-NAME              NAMESPACE        REVISION        UPDATED                                     STATUS          CHART                APP VERSION
-my-release        default          2               2021-09-06 16:18:40.021412 +0800 CST        deployed        milvus-2.1.14        2.0.0-rc.5-hotfix1
+$ helm repo update
+$ helm upgrade my-release milvus/milvus --set cluster.enabled=true
 ```
 
-You can see the new pods:
+Run `$ helm list` again to check your Milvus version. You can see your Milvus cluster has been updated to v2.0.0-rc8.
```
Comment (chris-zilliz) [inline site/en/adminGuide/upgrade.md:200]: updated or upgraded? Align, please.

**Pull request merged**